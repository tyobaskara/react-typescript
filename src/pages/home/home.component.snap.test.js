import renderer from 'react-test-renderer';
import Home from './home.component';

jest
  .mock('../../components/todo-list/todo-list.component', () => 'TodoListComponent');

describe('Home Snap Test', () => {
  test('should render correctly', () => {
    const component = renderer.create(
      <Home />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
