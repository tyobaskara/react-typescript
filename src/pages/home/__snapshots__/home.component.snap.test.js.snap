// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snap Test should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="group"
  >
    <h1>
      Homepage
    </h1>
  </div>
  <div
    className="group"
  >
    <h3>
      useContext
    </h3>
    <p>
      theme: 
      
    </p>
    <button
      onClick={[Function]}
    >
      Change Theme
    </button>
  </div>
  <br />
  <div
    className="group"
  >
    <h3>
      useState
    </h3>
    <p>
      Count: 
      1
    </p>
    <button
      onClick={[Function]}
    >
      Add Count
    </button>
  </div>
  <br />
  <div
    className="group"
  >
    <TodoListComponent />
  </div>
</div>
`;
